catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/GmdConverter.java

```diff
@@ -139,102 +141,207 @@ protected XmlTree buildTree(Set<Path> paths) {
-                metacard.getTitle()));
-
-        GregorianCalendar createdCal = new GregorianCalendar();
-
-        if (metacard.getCreatedDate() != null) {
-
-            createdCal.setTime(metacard.getCreatedDate());
-        }
-        createdCal.setTimeZone(UTC_TIME_ZONE);
-        pathValueTracker.add(new Path(GmdConstants.CREATED_DATE_PATH),
-                XSD_FACTORY.newXMLGregorianCalendar(createdCal)
-                        .toXMLFormat());
+        addDateAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.CREATED_DATE_PATH,
+                Core.CREATED);
 
         pathValueTracker.add(new Path(GmdConstants.CREATED_DATE_TYPE_CODE_PATH),
                 GmdConstants.METACARD_URI);
         pathValueTracker.add(new Path(GmdConstants.CREATED_DATE_TYPE_CODE_VALUE_PATH),
-                Metacard.CREATED);
-
-        pathValueTracker.add(new Path(GmdConstants.ABSTRACT_PATH), StringUtils.defaultString(
-                metacard.getDescription()));
-        pathValueTracker.add(new Path(GmdConstants.POINT_OF_CONTACT_PATH),
-                StringUtils.defaultString(metacard.getPointOfContact()));
-
-        pathValueTracker.add(new Path(GmdConstants.POINT_OF_CONTACT_ROLE_PATH), (String) null);
-
-        pathValueTracker.add(new Path(GmdConstants.LANGUAGE_PATH), StringUtils.defaultIfEmpty(
-                Locale.getDefault()
-                        .getLanguage(),
-                Locale.ENGLISH.getLanguage()));
-        addExtent(metacard, pathValueTracker);
+                Core.CREATED);
+
+        addListAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.ABSTRACT_PATH,
+                Core.DESCRIPTION);
+
+        addListAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.POINT_OF_CONTACT_PATH,
+                Contact.PUBLISHER_NAME);
+
+        addListAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.POINT_OF_CONTACT_EMAIL_PATH,
+                Contact.PUBLISHER_EMAIL);
+
+        addListAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.POINT_OF_CONTACT_PHONE_PATH,
+                Contact.PUBLISHER_PHONE);
+
+        addListAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.POINT_OF_CONTACT_ADDRESS_DELIVERY_POINT_PATH,
+                Contact.PUBLISHER_ADDRESS);
+
+        addListAttributeToXml(metacard, pathValueTracker, GmdConstants.KEYWORD_PATH, Topic.KEYWORD);
+
+        Attribute language = metacard.getAttribute(Core.LANGUAGE);
+        if (language != null) {
+            addListAttributeToXml(metacard,
+                    pathValueTracker,
+                    GmdConstants.LANGUAGE_PATH,
+                    Core.LANGUAGE);
+        } else {
+            pathValueTracker.add(new Path(GmdConstants.LANGUAGE_PATH), "eng");
```

Comment:
We should use Locale for this  https://docs.oracle.com/javase/7/docs/api/java/util/Locale.html#getISO3Country()